android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -135,11 +135,18 @@ public void setSizes(ReactNativeAdView reactViewGroup, ReadableArray value) {
       payload.putDouble("height", adSize.getHeight());
       sendEvent(reactViewGroup, EVENT_SIZE_CHANGE, payload);
     }
-
+    reactViewGroup.setSizesArray(value);
     reactViewGroup.setSizes(sizeList);
     reactViewGroup.setPropsChanged(true);
   }
 
+  @ReactProp(name = "maxAdHeight")
```

Comment:
I wanted to call the native prop "maxHeight" but that ends up setting the native view height too, which isn't what you want in the case where the user sets a height lower than the minimum (32px) because you get a minimum height ad back and then the view crops it.